rust/sbp/src/messages/invalid.rs

```diff
@@ -0,0 +1,131 @@
+//! Invalid messages, maybe don't have enough bytes are Crc Errors.
+
+use bytes::{Buf, BufMut};
+
+use crate::{de::CrcError, messages::SbpMsgParseError, wire_format::WireFormat, SbpMessage};
+
+/// The message returned by the parser when the message type does not correspond to a known message.
```

Comment:
Or if the CRC is invalid?

Reply:
I forgot to finish up this module's documentation. let me re-write it rq.